lib/ansible/plugins/filter/pow.yml

```diff
@@ -3,18 +3,18 @@ DOCUMENTATION:
   version_added: "1.9"
   short_description: power of (math operation)
   description:
-    - math operation that returns the N power of inputed number X ^ N
+    - Math operation that returns the Nth power of inputed number, C(X ^ N).
   notes:
-    - This is a passthrough to Python's C(itertools.product)
+    - This is a passthrough to Python's C(math.pow).
```

Comment:
Flagging this as well as a substantive change.

Reply:
I think this was a copy'n'paste artefact.